lmfdb/groups/abstract/web_groups.py

```diff
@@ -2386,6 +2389,11 @@ def _full(self):
     def sub(self):
         S = self._lookup(self.subgroup, self._full, WebAbstractGroup)
         # We set various properties from S for create_boolean_subgroup_string
+        if not S:
+            order = self.subgroup_order
+            self.order = order
+            self.pgroup = len(ZZ(order).abs().factor())==1
+            return self
```

Comment:
I think it might make more sense to create a new `WebAbstractGroup` object, so that the return type doesn't depend on whether it's in the database or not.

Reply:
I imagine there is an elegant way to do that, but I tried several variations and keep getting errors.